sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/aad/AADOAuth2AutoConfiguration.java

```diff
@@ -77,19 +77,21 @@ private ClientRegistration azureClientRegistration() {
         Assert.doesNotContain(tenantId, "/", "azure.activedirectory.tenant-id should not contain '/'.");
 
         List<String> scope = aadAuthenticationProperties.getScope();
-        if (aadAuthenticationProperties.allowedGroupsConfigured()
-            && !scope.contains("https://graph.microsoft.com/user.read")
-        ) {
-            scope.add("https://graph.microsoft.com/user.read");
-            LOGGER.warn("scope 'https://graph.microsoft.com/user.read' has been added.");
-        }
-        if (!scope.contains("openid")) {
-            scope.add("openid");
-            LOGGER.warn("scope 'openid' has been added.");
-        }
-        if (!scope.contains("profile")) {
-            scope.add("profile");
-            LOGGER.warn("scope 'profile' has been added.");
+        if (!scope.toString().contains(".default")) {
```

Comment:
Is it possible that one of the passed scopes contains substring like ".default", e.g. "my.default"?

Reply:
I manually searched all permissions provided in Azure portal, and haven't found there are any scopes contained ".default" in their scope names.